docs/details/signal.dox

```diff
@@ -171,15 +186,17 @@ functions can perform a 2D convolution on 3D signal and filters.
 
 \copydoc signal_func_conv_desc
 
-For three dimensional inputs with m, n & p sizes along the 0th, 1st & 2nd dimensions
+For three dimensional inputs with m, n & p sizes along the 0th, 1st & 2nd axes
 respectively, given below are the possible batch operations.
 
-| Input Signal Dimensions   | Filter Dimensions   | Output Dimensions   | Batch Mode                | Description                                                |
-| :-----------------------: | :-----------------: | :-----------------: | :-----------------------: | :------------                                              |
-| [m n p 1]                 | [a b c 1]           | [m n p 1]           | No Batch                  | Output will be a single convolve array                     |
-| [m n p 1]                 | [a b c d]           | [m n p d]           | Filter is Batched         | d filters applied to same input                            |
-| [m n p q]                 | [a b c 1]           | [m n p q]           | Signal is Batched         | 1 filter applied to q inputs                               |
-| [m n p k]                 | [a b c k]           | [m n p k]           | Identical Batches         | k filters applied to k inputs in one-to-one correspondence |
+| Signal Size        | Filter Size        | Output Size        | Batch Mode         | Description                                                |
+| :----------------: | :----------------: | :----------------: | :----------------: |:-----------------------------------------------------------|
+| \dims{m, n, p, 1}  | \dims{a, b, c, 1}  | \dims{m, n, p, 1}  | No Batch           | Output will be a single convolve array                     |
```

Comment:
nit: Let's be consistent whether we use capital letters for the dims or small letters

Reply:
This `dims` has nothing to do with textual references. It is command alias and we are good with `\dims{}` .